livox_ros2_driver/livox_ros2_driver/lddc.cpp

```diff
@@ -746,11 +747,17 @@ void Lddc::checkTemperature(diagnostic_updater::DiagnosticStatusWrapper & stat)
       level = DiagStatus::ERROR;
     }
 
+    if (level != DiagStatus::OK) {
+      if (error_str != "") error_str += ", ";
+      error_str += broadcast_code + ": " + temperature_dict_.at(level);
```

Comment:
I feel it's better to add as key-values, not as a message.

Reply:
Also, we should change `hardware_id` instead.

Reply:
Do you mean it's better to split it into multiple diagnostics like https://github.com/tier4/AutowareArchitectureProposal.iv/blob/3bc4bc67714bb27187c756b98880a77229e58bd2/system/autoware_state_monitor/src/autoware_state_monitor_node/diagnostics.cpp#L33-L41?

Reply:
Yes, it's better.
(Ah, I'm sorry I misunderstood. You can't change `hardware_id` dynamically with `diagnostic_updater`.)

Reply:
But if you open an issue, I think your implementation is tentatively acceptable.

Reply:
I updated to run a diagnostics for each device.